[IMP] Inventory: add delivery slip setting to lots (#11698)
odoo/documentation: content/applications/inventory_and_mrp/inventory/product_management/product_tracking/lots.rst

.. _inventory/product_management/lot-traceability:

Lots on delivery slips
======================

When selling products tracked using lots, it is possible to include those lot numbers on the
delivery slips included in packages delivered to customers. This can be helpful to customers in
cases where they need the lot number to file an RMA or repair request, or to register the product
for any reason.

To include lot numbers on delivery slips, open the :menuselection:`Inventory` app, and navigate to
:menuselection:`Configuration -> Settings`. Scroll down to the :guilabel:`Traceability` section,
tick the checkbox next to :guilabel:`Display Lots & Serial Numbers on Delivery Slips` and click
:guilabel:`Save`.

> 

Reviewer: Do you want to mention the 'Display Expiration Date s in Delivery Slips' that appears when this option is ticked? I assume expiration dates can be on lots, so that might be good to mention - even just in a note field.

Author: That option is only available in 18.1. Once this PR is merged, I will push an update PR specifically targeting that branch.

After enabling the :guilabel:`Display Lots & Serial Numbers on Delivery Slips` setting, lot numbers
automatically appear on delivery slips for every product tracked using lot numbers.

> To view a delivery slip, navigate to the :menuselection:`Inventory` app, click on the
> :guilabel:`Delivery Orders` card, and select a delivery order. Click :guilabel:`Validate` to confirm
> the delivery and add product information to the delivery slip.

Reviewer: ```suggestion
To view a delivery slip with visible lot numbers, navigate to the default :menuselection:`Inventory` app dashboard,  or click :menuselection:`Inventory` app > Overview`.  Click 
:guilabel:`Delivery Orders`  on the corresponding card, and select a delivery order. Click :guilabel:`Validate` to confirm
the delivery and add product information to the delivery slip.
```
I wanted to add instructions for if you were not on the dashboard view - I was previously in the settings, so just adding the'click Overview' to let them know where to find it. Also, you must click on the text, not anywhere int he box, so I called that out.

I also noticed on the receipt I clicked on, there were no lot numbers. I can only add 'Serial Numbrs' to the view, not 'Lot Numbers'. Is this where the elot number would be seen? If so, I would mention to make the field visible, and that it appears there. 

And, you could mention you must select an order with lot numbers - I clicked a random one, so I don't know if there even is a lot number for it,

Author: The column is called "Serial Numbers" but it displays lot numbers as well.

At the top of the order's form, click the :icon:`fa-cog` :guilabel:`(Actions)` button, and select
:guilabel:`Print -> Delivery Slip`. The delivery slip is then downloaded. Open the delivery slip
using the device's browser or file manager. Lot numbers are listed next to their respective products

> in :guilabel:`Lot/Serial Number` column of the order lines section.

Reviewer: ```suggestion
in the :guilabel:`Lot/Serial Number` column of the order lines section.
```

> .. image:: lots/delivery-slip.png

Author: I've added a paragraph about how to see the lot number on the order before printing the slip.

.. _inventory/product_management/lot-traceability:

Display lots on delivery slips
==============================

When selling products tracked with lots, it is possible to include the lot numbers on the delivery
slips sent to customers. This can be helpful to customers in cases where lot numbers are needed,
such as filing an RMA or repair request, or registering the product.

To include lot numbers on delivery slips, open the :menuselection:`Inventory` app, and navigate to
:menuselection:`Configuration --> Settings`. Scroll down to the :guilabel:`Traceability` section,
tick the :guilabel:`Display Lots & Serial Numbers on Delivery Slips` checkbox, and click
:guilabel:`Save`.

After enabling the :guilabel:`Display Lots & Serial Numbers on Delivery Slips` setting, lot numbers
are listed on delivery slips for products tracked by lots, once the delivery order is validated.

To view lot numbers on delivery orders and delivery slips, navigate to the
:menuselection:`Inventory` app, click on :guilabel:`Delivery Orders`, and select an order containing
a product tracked using lots.

To view the lot numbers of products included in the order, make sure the :guilabel:`Operations` tab

> is selected, then click the :icon:`oi-settings-adjust` (adjust) button to the right of the tab.
> Ensure that the :guilabel:`Serial Numbers` checkbox is ticked, which causes a :guilabel:`Serial
> Numbers` column to appear. The lot number(s) for each product included in the order are displayed in
> this column.

Reviewer 2: Please add a :guilabel: to the icon descriptor :)
```suggestion
is selected, then click the :icon:`oi-settings-adjust` :guilabel:`(adjust)` button to the right of
the tab. Ensure that the :guilabel:`Serial Numbers` checkbox is ticked, which causes a
:guilabel:`Serial Numbers` column to appear. The lot number(s) for each product included in the
order are displayed in this column.
```

.. _inventory/product_management/lot-traceability:

Display lots on delivery slips
==============================

When selling products tracked with lots, it is possible to include the lot numbers on the delivery
slips sent to customers. This can be helpful to customers in cases where lot numbers are needed,
such as filing an RMA or repair request, or registering the product.

To include lot numbers on delivery slips, open the :menuselection:`Inventory` app, and navigate to

> :menuselection:`Configuration -> Settings`. Scroll down to the :guilabel:`Traceability` section,

Reviewer 3: ```suggestion
:menuselection:`Configuration --> Settings`. Scroll down to the :guilabel:`Traceability` section,
```

> tick the checkbox next to :guilabel:`Display Lots & Serial Numbers on Delivery Slips`, and click

Reviewer 3: ```suggestion
tick the :guilabel:`Display Lots & Serial Numbers on Delivery Slips` checkbox, and click
```

:guilabel:`Save`.

After enabling the :guilabel:`Display Lots & Serial Numbers on Delivery Slips` setting, lot numbers
automatically appear on delivery slips for every product tracked using lot numbers.

To view lot numbers on delivery orders and delivery slips, navigate to the
:menuselection:`Inventory` app, click on :guilabel:`Delivery Orders`, and select an order containing
a product tracked using lots.

> To view lot numbers on the order itself, make sure the :guilabel:`Operations` tab is selected, then
> click the :icon:`oi-settings-adjust` (additional options) button to the right of the tab. Ensure
> that the :guilabel:`Serial Numbers` checkbox is ticked, which causes a :guilabel:`Serial Numbers`
> column to appear. The lot number(s) for each product included in the order are displayed in this
> column.

Reviewer 3: This is great work incorporating Lara's advice to include details on assigning lot numbers to a product! However, this information doesn’t directly pertain to the instructions for adding lot numbers onto delivery slips. Since there are three different ways to assign lot numbers and you’ve detailed only one, I suggest placing this in a note admonition block for clarity and relevance.

Also, as per [Jess's icon guide](https://www.odoo.com/odoo/knowledge/17087), the name we've settled on as a team for that settings adjust icon is "adjust".

![image](https://github.com/user-attachments/assets/e05ff189-9304-4325-bc26-3b7e86fd10eb)

Here's a revised suggestion for the note block:

```suggestion
.. note::
   One of the [three ways to assign lot numbers on a delivery order]_(link to Liz's lot number doc) is by specifying it in the :guilabel:`Serial Numbers` field in the :guilabel:`Operations` tab. If that field is not visible by default, click the :icon:`oi-settings-adjust` :guilabel:`(adjust)` icon and tick the :guilabel:`Serial Numbers` checkbox.
```

When the order is ready to be processed, click :guilabel:`Validate` to confirm the delivery and add
product information to the delivery slip.

At the top of the order's form, click the :icon:`fa-cog` :guilabel:`(Actions)` button, and select
:guilabel:`Print -> Delivery Slip`. The delivery slip is then downloaded. Open the delivery slip
using the device's browser or file manager. Lot numbers are listed next to their respective products
in the :guilabel:`Lot/Serial Number` column.

.. image:: lots/delivery-slip.png

>    :align: center

Reviewer 3: ```suggestion
```
